spec/system/candidate_interface/entering_details/candidate_entering_personal_details_when_requiring_visa_spec.rb

```diff
@@ -3,17 +3,16 @@
 RSpec.describe 'Entering personal details' do
   include CandidateHelper
 
+  around do |example|
+    Timecop.freeze(2021, 7, 6, 12) { example.run }
```

Comment:
Can we use  `Timecop.freeze(RecruitmenyCycle.current_year, 7, 6, 12) { example.run }` please just so it doesn't go stale in a year or twos time.